lib/DDG/Spice/DebVersion.pm

```diff
@@ -1,9 +1,18 @@
-# Get the version of a package in various debian repositories
 package DDG::Spice::DebVersion;
 
```

Comment:
@javathunderman I added that string to the `description` metadata. Should I add it again as ABSTRACT?